### ISSUE TITLE
Issues with the build process

### ISSUE THREAD

**Issue by OriginalEXE** [moxiecode/moxie#142]
Hi,

first of all, thanks for your work, awesome library you have here.

I was building a custom moxie build today as I don't need all the runtimes, and have ran into issues.

It seems like there is an issue with the amdlc module you have included. For some reason, each time I tried running the build, it threw error about compileMinified method being undefined (in Jakefile.js).

What I had to do to fix this is uninstall your default amdlc module, change the git repo in package.json for it to original: https://github.com/spocke/amdlc/tarball/master, and then reinstall hte module.

This caused another issue though, AmdCompiler from amdlc/lib thre errors about currentReporter being undefined. Fortunatelly, it is just for notifications, so when I commented out each line containing call to currentReporter.debug, the build succeeded.

So basically this is not urgent, as I was able to get it working, but it might be worth looking into.

I was building on Ubuntu.

Thanks

**Comment by jayarjo** [comment]
Thanks for letting us know! We have eventually fixed that out in the build script.
